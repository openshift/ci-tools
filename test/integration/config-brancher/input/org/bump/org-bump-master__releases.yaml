base_images:
  tools:
    name: "4.5"
    namespace: ocp
    tag: tools
base_rpm_images:
  installer:
    name: "4.5"
    namespace: ocp
    tag: installer
rpm_build_commands: make rpms
build_root:
  image_stream_tag:
    name: "4.5"
    namespace: ocp
    tag: base
promotion:
  name: "4.5"
  namespace: ocp
releases:
  initial:
    integration:
      name: "4.5"
      namespace: ocp
  latest:
    integration:
      include_built_images: true
      name: "4.5"
      namespace: ocp
tests:
  - as: unit
    commands: make test-unit
    container:
      from: src
resources:
  '*':
    limits:
      cpu: 500Mi
    requests:
      cpu: 10Mi
zz_generated_metadata:
  branch: master
  org: org
  repo: bump
  variant: releases
