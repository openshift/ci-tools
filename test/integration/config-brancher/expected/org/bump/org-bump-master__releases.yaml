base_images:
  tools:
    name: "4.6"
    namespace: ocp
    tag: tools
base_rpm_images:
  installer:
    name: "4.6"
    namespace: ocp
    tag: installer
build_root:
  image_stream_tag:
    name: "4.6"
    namespace: ocp
    tag: base
promotion:
  name: "4.6"
  namespace: ocp
releases:
  initial:
    integration:
      name: "4.6"
      namespace: ocp
  latest:
    integration:
      include_built_images: true
      name: "4.6"
      namespace: ocp
resources:
  '*':
    limits:
      cpu: 500Mi
    requests:
      cpu: 10Mi
rpm_build_commands: make rpms
tests:
- as: unit
  commands: make test-unit
  container:
    from: src
zz_generated_metadata:
  branch: master
  org: org
  repo: bump
  variant: releases
